interface/ceed-basis.c

```diff
@@ -203,7 +203,10 @@ static int CeedBasisCreateProjectionMatrices(CeedBasis basis_from, CeedBasis bas
   // Check for compatible quadrature spaces
   CeedCall(CeedBasisGetNumQuadraturePoints(basis_to, &Q_to));
   CeedCall(CeedBasisGetNumQuadraturePoints(basis_from, &Q_from));
-  CeedCheck(Q_to == Q_from, ceed, CEED_ERROR_DIMENSION, "Bases must have compatible quadrature spaces");
+  CeedCheck(Q_to == Q_from, ceed, CEED_ERROR_DIMENSION,
+            "Bases must have compatible quadrature spaces."
+            " 'basis_from has %" CeedInt_FMT " points and 'basis_to' has %" CeedInt_FMT,
```

Comment:
insignificant typo:

```suggestion
            " 'basis_from' has %" CeedInt_FMT " points and 'basis_to' has %" CeedInt_FMT,
```

Thanks for the improvements!

Reply:
Ugh, lemmie get that into a new PR, thanks!

Reply:
https://github.com/CEED/libCEED/pull/1635